### PR TITLE
Change secToDate returns and typo

### PR DESCRIPTION
**Pull request by gnituy18**

### CODE REVIEW

File: src/libs/__tests__/Dad.spec.js [hunk 1]
```diff
@@ -41,12 +41,36 @@ describe('Dad functionality', () => {
...
+      month: 0,
+      date: 1,
+      hours: 1,
+      minutes: 0,
+      seconds: 0,
+      day: 1,
+    });
+    expect(indo(UTC0101)).toEqual({
+      year: 2017,
+      month: 11,
```
Comment (rwu823) [inline src/libs/__tests__/Dad.spec.js]: @gnituy18  不是要改成 12 嗎?

File: src/libs/__tests__/Dad.spec.js [hunk 2]
```diff
@@ -41,12 +41,36 @@ describe('Dad functionality', () => {
       .toThrowError('Invaild date string format');
   });
 
-  it('if inpunt is second number', () => {
-    const UTC0105 = new Date('2018-01-05 00:00:00+00:00') / 1000;
+  it('if input is second number', () => {
+    const UTC0101 = new Date('2018-01-01 00:00:00+08:00') / 1000;
```
Comment (rwu823) [inline src/libs/__tests__/Dad.spec.js]: It's `+8` the variable's naming  should be `CST0101` would be more clearly.

File: src/libs/__tests__/Dad.spec.js (hunk 1 above)
Comment (gnituy18) [reply src/libs/__tests__/Dad.spec.js]: 後來覺得保持與date用法一致會比較不容易出錯

File: src/libs/__tests__/Dad.spec.js (hunk 2 above)
Comment (gnituy18) [reply src/libs/__tests__/Dad.spec.js]: [fixed](https://github.com/17media/dad/pull/14/files#diff-0a42e525241c46c84c5b08c85f34aaf4R45)

**Pull request merged**